js/compiler/compiler.go

```diff
@@ -34,6 +37,25 @@ import (
 	"go.k6.io/k6/lib"
 )
 
+//nolint:gochecknoglobals
+var maxSrcLenForBabelSourceMap = 250 * 1024
+
+const maxSrcLenForBabelSourceMapVarName = "K6_DEBUG_SOURCEMAP_FILESIZE_LIMIT"
+
+//  drop this code and everything it's connected to when goja is dropped
+func init() {
+	v := os.Getenv(maxSrcLenForBabelSourceMapVarName)
+	if len(v) > 0 {
+		i, err := strconv.Atoi(v)
+		if err != nil {
+			//nolint:forbidigo
+			fmt.Printf("Tried to parse %q from %s as integer but couldn't %s\n",
```

Comment:
At least use `log` here, since we redirect its output to our own logger in https://github.com/grafana/k6/blob/13b2372fd98fb3fefe103a1d22f212164dff9107/cmd/root.go#L138

Reply:
doesn't really matter as this happens in hte `init` before that line is called

Reply:
I can add one more `sync.Once` at this point :shrug:

Reply:
:+1: for either another `sync.Once` instead of `init()`, or for just silently ignoring any parse error